UPGRADE-14.5-tech.md

```diff
@@ -132,3 +132,58 @@ The releases of Shopsys Platform adhere to the [Backward Compatibility Promise](
 For commands not covered by Make targets (e.g., `exec`, `logs`, `restart`), use plain `docker compose` like `docker compose exec php-fpm bash`
 
 -   see #project-base-diff to update your project
+
+#### stop processing images by PHP to avoid decreasing quality ([#3169](https://github.com/shopsys/shopsys/pull/3169))
+
+-   `Shopsys\FrameworkBundle\Component\Image\Processing\ImageProcessor` class was changed:
+    -   all the methods and constants are now strictly typed
+    -   `__construct()` method changed its interface:
+        ```diff
+            public function __construct(
+        -        protected readonly ImageManager $imageManager,
+                 protected readonly FilesystemOperator $filesystem,
+        ```
```

Comment:
As I can live with the current state (as we can copy-paste those, right?), we could maybe rather update the general section regarding the strict types and general changes easily spotted by static analysis, as we did in the later upgrading notes.

The thing is, now we're just cherry-picking existing changes, but if we implemented something new (maybe Symfony 7 upgrade) in the `14.5` version first, we would have to write the upgrade notes the old way to keep it consistent :shrug: